City-of-Helsinki/hitas
# Max price breakdown

# Hitas Pull Request

# Description

- Frontend and PDF breakdowns of maximum price calculations for all pre-2011 cases
- Some bugfixes/refactoring

## Pull request checklist

Check the boxes for each DoD items that has been completed:

- **Frontend**
- **Backend**
    - [x] Automatic tests has been added
    - [ ] Database migrations will work in test environment

## Test plan

- Create a new max price calculation on a pre-2011 apartment
  - Check out the new and amazing breakdowns for all different indices
- Confirm the max price calculation
  - Download the PDF for max price
    - Be amazed at how beautiful it is

## Tickets

This pull request resolves all or part of the following ticket(s): HT-315, HT-340, HT-353
